src/components/drs/Drs.js

```diff
@@ -3,72 +3,291 @@ import axios from 'axios';
 import React from 'react';
 import {
   Switch,
-  Route
+  Route,
+  Redirect
 } from "react-router-dom";
 import { Typography } from '@material-ui/core';
+import { format } from 'date-fns';
 import DrsIndex from './pages/DrsIndex';
 import DrsShow from './pages/DrsShow';
+import NewDrs from './pages/NewDrs';
 
-const cancelToken = axios.CancelToken;
-const drsCancelToken = cancelToken.source();
+let newDate = new Date();
```

Comment:
Do you need to unpack all attributes of a Date for the constructor below? Why not just do:

```
let newDate = new Date();
newDate.setSeconds(0, 0);
...
created_time: format(newDate, "yyyy-MM-dd'T'HH:mm:ss'Z'"),
```